paywall/src/components/interface/Logo.js

```diff
@@ -1,37 +1,34 @@
-import PropTypes from 'prop-types'
 import styled from 'styled-components'
 import React from 'react'
 import Unlock from './svg/Unlock'
 import UnlockWordMark from './svg/UnlockWordMark'
+import Media from '../../theme/media'
 
 /**
  * This wraps the Unlock SVG component into a pink circle.
  * Note: we need to keep the viewBox to ensure that the SVG will resize.
  * @param {string} size
  */
-export const RoundedLogo = ({ size }) => (
```

Comment:
note: this configurability is removed for 2 reasons:

1. the rounded logo is only used in locked, unlocked and optimistic unlocking flags
2. The size is not responsive, and assumes desktop view.

The current solution is far more responsive. If, in the future, a need for a larger rounded logo appears, then the feature can be easily added back in. As of now, it is not needed and makes using it harder.